.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # https://help.github.com/articles/about-codeowners/
 
-* @ikhoon @linxGnu @minwoox @trustin
+* @ikhoon @jrhee17 @linxGnu @minwoox @trustin
```

Comment:
Double checked that the handle exists, which means write access must be missing for https://github.com/linxGnu

Reply:
Oopsie! Let me remove him.